### PR TITLE
Implement filterdup function

### PR DESCRIPTION
**Pull request by hubentu**

### CODE REVIEW

File: R/install_MACS.R [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+#'     virtualenv_list use_virtualenv conda_create conda_install
+#'     use_condaenv
+#' @export
+
+install_MACS <- function(envname = "MACS", method = "conda") {
+    stopifnot(is.character(envname) && length(envname) == 1)
+    is_windows <- identical(.Platform$OS.type, "windows")
+    is_osx <- Sys.info()["sysname"] == "Darwin"
+    is_linux <- identical(tolower(Sys.info()[["sysname"]]), "linux")
+    if (!is_windows && !is_osx && !is_linux) {
```
Comment (taoliu) [inline R/install_MACS.R:17]: I don't think MACS can be installed in windows through conda.

File: tests/testthat/test.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    def __init__(self, **kwargs):
+        self.__dict__.update(kwargs)
+
+options = Namespace(gsize = 2.7e9,
+                    format = "BEDPE", 
+                    keepduplicates = "1",
+                    verbose = 1,
+                    outputfile = "CTCF_ChIP_200K_filterdup.bed",
+                    outdir = "/tmp",
+                    ifile = ["/Users/qi28068/Workspace/MACSr/tests/testthat/CTCF_PE_CTRL_chr22_50k.bedpe.gz"],
```
Comment (taoliu) [inline tests/testthat/test.py:16]: Let's include the testing files in the package. You can use the files under https://github.com/macs3-project/MACS/tree/master/test

**Review by taoliu — COMMENTED** [review]
Please see the two comments. Also, what's zzz.R file for?

File: R/install_MACS.R (hunk 1 above)
Comment (hubentu) [reply R/install_MACS.R:17]: OK. I will remove windows.

File: tests/testthat/test.py (hunk 2 above)
Comment (hubentu) [reply tests/testthat/test.py:16]: Will add test in the next pull

File: R/install_MACS.R (hunk 1 above)
Comment (hubentu) [reply R/install_MACS.R:17]: zzz.R is a script to run when loading the package. For now, it only checks whether macs is installed.

**Pull request merged**